### PR TITLE
Add a statsd helper. #480

### PR DESCRIPTION
**Pull request by greeshmab**
Adding a statsd helper function to get the status code

### CODE REVIEW

File: cliquet/statsd.py [hunk 1]
```diff
@@ -33,6 +33,10 @@ def count(self, key, unique=None):
         else:
             return self._client.set(key, unique)
 
+def status_code(request,str):
+	statsd=request.registry.statsd
+	if statsd:
+		statsd.count(str);
```
Comment (Natim) [inline cliquet/statsd.py]: Please use 4 spaces rather than tabulation in your code.

File: cliquet/statsd.py [hunk 2]
```diff
@@ -33,6 +33,10 @@ def count(self, key, unique=None):
         else:
             return self._client.set(key, unique)
 
+def status_code(request,str):
+	statsd=request.registry.statsd
```
Comment (Natim) [inline cliquet/statsd.py]: nit missing space around ``=`` operator

File: cliquet/statsd.py [hunk 3]
```diff
@@ -33,6 +33,10 @@ def count(self, key, unique=None):
         else:
             return self._client.set(key, unique)
 
+def status_code(request,str):
```
Comment (Natim) [inline cliquet/statsd.py]: ``status_code`` should be rename ``statsd_count``

``str`` is a reserved keyword in python, please use ``count_key`` instead.

You need a space betwee ``,`` and ` str``

**Comment by Natim** [comment]
Thank you for your work on this @greeshmab.
Could you please work on a test to make sure that it works if statsd is not defined and that if it is defined, statsd is called with the right key?

**Comment by ametaireau** [comment]
Thanks for taking the time to write a patch. It still needs some work in order to be accepted!

**Comment by Natim** [comment]
Superseeded by #574

**Pull request closed without merging**